### PR TITLE
Added a Makefile for building b2gremote.xpi from src/

### PR DESCRIPTION
**Pull request by emilis**
Added both a `Makefile` and `.gitignore`.

One can use:

```
$ make
```

inside project directory to build `b2gremote.xpi`.

Prerequisites: _make_, _zip_.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.swp
```
Comment (fabricedesre) [inline .gitignore]: Why do you need to ignore that?

Comment (emilis) [inline .gitignore]: ViM swap files :)

Added out of habit.

On 5/15/13, Fabrice Desré notifications@github.com wrote:

> > @@ -0,0 +1,2 @@
> > +*.swp
> 
> Why do you need to ignore that?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/fabricedesre/b2gremote/pull/7/files#r4227186

## 

Sent from my mobile device

Emilis Dambauskas

emilis.d@gmail.com
gsm: +370-686-07732
irc: emilis_info@irc.freenode.net
http://emilis.info

-----BEGIN GEEK CODE BLOCK-----
Version: 3.12
GAT/CC/MC/O dpu(-) s:- a- C++ UBLHS++ P(+) L+++ E--- W+++$ N+ o-- K?
!w O? M-@ V? PS+(--) PE Y+>++ PGP t- 5? X+@ R- !tv b+ DI++++ D G e++
h---- r+++ y++++
------END GEEK CODE BLOCK------

Comment (fabricedesre) [inline .gitignore]: Well, please remove that and I'll merge!

**Comment by emilis** [comment]
Updated pull request as per your request.

**Pull request merged**